MEDO v0.5.owl

```diff
@@ -502,6 +575,52 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
     
 
 
+    <!-- http://purl.obolibrary.org/obo/MEDO_0000101 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/MEDO_0000101">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/medo.owl/MEDO_0000100"/>
+        <obo:IAO_0000115>A formal process of evaluating the degree to which a student or teacher has achieved one or more learning or skill objectives. This process coheres with standards-aligned methods of evaluation and returns an official valuation of performance.</obo:IAO_0000115>
```

Comment:
@JaronJCheung Most of your definitions do not adhere to the genus-species format, i.e. A =def a B that Cs, where "A" denotes the term defined, "B" denotes its parent, and "C" what distinguishes A from siblings. [IAO_0000115](http://purl.obolibrary.org/obo/IAO_0000115) is itself defined to be used this way. 

See [OBO Principle 6](https://obofoundry.org/principles/fp-006-textual-definitions.html)
See Building Ontologies with Basic Formal Ontology, pg. 69
See [Definitions in Ontologies, pg 5](https://philpapers.org/archive/SEPDIO.pdf)

I encourage you to keep definitions to a single, grammatical, sentence as much as possible. There are separate annotation properties available for comments, examples, etc. 

You've provided no bibliographic citation for the definition and you've used a label "formal assessment" that is ambiguous. I searched '"formal assessment" definition' and definitions returned were distinct from yours, e.g. some restricted to student assessment, some required assessment to be structured. "Formal assessment" is also used in psychology research, where it is focused on clinical assessments of patients. 

What are "standards-aligned methods"? What standards? How strict is "official" meant to be? If I homeschool Mira, create a curriculum, measure her progress based on some template I come up with, is this official? 

Are you drawing a distinction between learning and skill that tracks 'knowledge that' vs 'knowledge how'? 

By "returns" do you mean "output of"?